app/helpers/thumbnail_helper.rb

```diff
@@ -82,7 +82,7 @@ def thumbnail(image, args = {})
       notes: notes
     }.merge(args)
 
-    render "images/image_thumbnail", locals
+    render partial: "images/image_thumbnail", locals: locals
```

Comment:
Does the old line work., and would you prefer to use it?
```
    render "images/image_thumbnail", locals
```
My understanding `partial` is necessary only when passing something other than `locals` or when `locals` has >1 entry.

Reply:
That must be why it couldn't find the locals unless i added the keywords. There are a lot of them.